Marble Blast Platinum/platinum/server/scripts/modes.cs

```diff
@@ -236,8 +236,8 @@ class = %objName;
 	}
 	if (isObject(%this)) {
 		if (!%this.hasCallback[%callback]) {
-			if ($DEBUG)
-				echo("[Mode" SPC %this.name @ "] Unused Callback" SPC %callback);
+			//if ($DEBUG)
+				//echo("[Mode" SPC %this.name @ "] Unused Callback" SPC %callback);
```

Comment:
Nice, this has been bothering me as well. But I think the echo might still be useful in debugging in some cases. Maybe we should do:

```suggestion
			if ($DEBUG && !(%this.name $= "null" && %callback $= "onFrameAdvance"))
				echo("[Mode" SPC %this.name @ "] Unused Callback" SPC %callback);
```

Reply:
New gamemodes are basically never devved, but it miiiiiiiight be useful for custom MIS code. So the best solution would be to patch only the onFrameAdvance command. I think it should be patched for all gamemodes (because it's pretty obvious what it does lol).